futures-channel/Cargo.toml

```diff
@@ -10,8 +10,16 @@ description = """
 Channels for asynchronous communication using futures-rs.
 """
 
+[features]
+std = []
```

Comment:
Do we want an `std` feature here? In my last PR, I removed it from `futures-channel` since everything in `futures-channel` requires `std`.

Reply:
Oh I recalled you saying we'd just have `std` everywhere for consistency, but yeah I can remove this.

Reply:
Yeah, I said that's why I did it originally and then I removed it haha

Reply:
I'm fine w/ either. If we're including the feature, we should if_std out the body of futures_channel